demo/src/app/components/+collapse/demos/toggle-manual/toggle-manual.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'toggle-manual-demo',
+  templateUrl: './toggle-manual.component.html'
+})
+export class ToggleManualDemoComponent {
+  showSelected = true;
+  hideContent() {
+    this.showSelected = true;
```

Comment:
show / hide must be done without ngIf, using `hide()` and `show()` methods of `CollapseDirective`